keps/sig-network/784-kube-proxy-component-config/README.md

```diff
@@ -127,6 +131,33 @@ The mitigations to those risks:
 
 ## Design Details
 
+### Changes in structure
+
+| v1alpha1               | v1alpha2                | Comments                                                                    |
+|------------------------|-------------------------|-----------------------------------------------------------------------------|
+|                        | Linux                   | new section for linux (platform-specific) options                           |
+| Conntrack              | Linux.Conntrack         | moved from root to linux (platform-specific) section                        |
+| OOMScoreAdj            | Linux.OOMScoreAdj       | moved from root to linux (platform-specific) section                        |
+| IPTables.MasqueradeBit | Linux.MasqueradeBit     | moved from iptables (backend-specific) to linux (platform-specific) section |
+| NFTables.MasqueradeBit | Linux.MasqueradeBit     | moved from nftables (backend-specific) to linux (platform-specific) section |
+| IPTables.MasqueradeAll | Linux.MasqueradeAll     | moved from iptables (backend-specific) to linux (platform-specific) section |
+| NFTables.MasqueradeAll | Linux.MasqueradeAll     | moved from nftables (backend-specific) to linux (platform-specific) section |
+|                        | Windows                 | new section for windows (platform-specific) options                         | 
```

Comment:
It's possible that some of the options currently under `Winkernel` are actually windows-generic? (Check if the old winuserspace backend used any of the winkernel config?) Also, maybe the `--windows-service` CLI option that gets added from `cmd/kube-proxy/app/init_windows.go`

Reply:
We can add --windows-service. All other options are for configuring HNS only, cross-checked with old userspace windows proxy.